exercism/hamming/hamming.js

```diff
@@ -0,0 +1,27 @@
+var Hamming = function() {}
```

Comment:
Nice job on this one! The only 'suggestion' I can think of is to use single line if statements when there's only one line in the block. For example:
if( aString === bString ) return 0

However, I've been told that I overuse short-hand statements. I think it's more about readability, but in a couple cases in your code, it shouldn't affect it